plugins/catalog-react/api-report.md

```diff
@@ -144,6 +149,26 @@ export type DefaultEntityFilters = {
   namespace?: EntityNamespaceFilter;
 };
 
+// @public
+export function defaultEntityPresentation(
```

Comment:
Yeah the original idea was actually to enable its direct use. Note that this is not the same thing as the full renderer given to `DefaultEntityPresentationApi`. It's instead the fallback that you can use when all else fails, in either the default renderer or your own custom ones. It just tries to something vaguely useful and won't typically be the "main path" but rather something you fall back on.

Note that it's both used by [the catalog package](https://github.com/backstage/backstage/blob/10d83db0d203c1aa88c7c86ac24b4275833fe401/plugins/catalog/src/apis/EntityPresentationApi/DefaultEntityPresentationApi.ts#L167) and [the catalog-react package](https://github.com/backstage/backstage/blob/10d83db0d203c1aa88c7c86ac24b4275833fe401/plugins/catalog-react/src/apis/EntityPresentationApi/useEntityPresentation.ts#L64) for different purposes. If we make a default implementation of the API for tests, it'd probably be calling this one too.

Maybe it's the case that I should just move the `useEntityPresentation` to the catalog package instead of catalog-react? Then I think all usage is within the catalog package at least, so we can hide it until any actual need arises to expose it